### PR TITLE
Creating Issue templates (BSP-477)

### PR DESCRIPTION
**Pull request by Indastri**
Issue templates for esp-bsp repository

### CODE REVIEW

**Comment by espzav** [comment]
@Indastri Thank you for this PR. Please, how can I test it before merging?

**Review by espzav — APPROVED** [review]
@Indastri Good work! It looks good.

File: .github/ISSUE_TEMPLATE/02_Issue-Report.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      description: Is anything else attached to the development board?
+      placeholder: ex. GPIO 18 & 19 are connected to I2C devices.
+    validations:
+      required: true
+  - type: input
+    id: IDE
+    attributes:
+     label: IDE Name
+     description: What IDE are you using?
+     placeholder: eg. Arduino IDE, PlatformIO, Sloeber...
```
Comment (VojtechBartoska) [inline .github/ISSUE_TEMPLATE/02_Issue-Report.yml:40]: please update placeholders, BSP is IDF based so Arduino IDE does not make a lot of sense to be listed here.

File: .github/ISSUE_TEMPLATE/02_Issue-Report.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  - type: textarea
+    id: other-remarks
+    attributes:
+      label: Other Steps to Reproduce 
+      description: Is there any other information you can think of which will help us reproduce this problem? Any additional info can be added as well.
+      placeholder: ex. I also tried on other OS, HW...it works correctly on that setup.
+  - type: checkboxes
+    id: confirmation
+    attributes:
+      label: I have checked existing issues, online documentation and the Troubleshooting Guide
```
Comment (VojtechBartoska) [inline .github/ISSUE_TEMPLATE/02_Issue-Report.yml:77]: We do not have that much online documentation (there is some content in IDF programming guide, maybe we cna link it somewhere) or any Troubleshooting guide, there is just contribution guide so please update this.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1 @@
+blank_issues_enabled: true
```
Comment (VojtechBartoska) [inline .github/ISSUE_TEMPLATE/config.yml:1]: what about adding here link to documentation and link to esp32 forum?

**Review by VojtechBartoska — CHANGES_REQUESTED** [review]
Thanks for PR, added few comments.

File: .github/ISSUE_TEMPLATE/01_Feature-Request.yml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    attributes:
+      value: |
+        * We welcome any ideas or feature requests! It is helpful if you can explain exactly why the feature would be useful.
+        * There are usually some outstanding feature requests in the [Existing issues list](https://github.com/espressif/esp-bsp/issues?q=is%3Aopen+is%3Aissue+label%3A%22Type%3A+Feature+request%22), feel free to add comments to them.
+  - type: input
+    id: Area
+    attributes:
+      label: Related area
+      description: Please briefly explain the area of your Feature Request.
+      placeholder: eg. Display, peripheral, etc.
```
[the remark this answers is not in the document]
Comment (Indastri) [reply .github/ISSUE_TEMPLATE/01_Feature-Request.yml]: Updated

File: .github/ISSUE_TEMPLATE/config.yml (hunk 3 above)
Comment (Indastri) [reply .github/ISSUE_TEMPLATE/config.yml:1]: links added

**Comment by VojtechBartoska** [comment]
> @Indastri Thank you for this PR. Please, how can I test it before merging?

you can check it here: https://github.com/Indastri/esp-bsp/issues/new/choose

File: .github/ISSUE_TEMPLATE/config.yml [hunk 5]
```diff
@@ -0,0 +1,3 @@
...
+[README.md](../README.md)
+[ESP32 forum](https://www.esp32.com/)
```
Comment (VojtechBartoska) [inline .github/ISSUE_TEMPLATE/config.yml:3]: I would suggest adding here structure with Description and link.

For example:

```
contact_links:
  - name: BSP Repository README
    url: https://github.com/espressif/esp-bsp/blob/master/README.md
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 6]
```diff
@@ -0,0 +1,3 @@
+blank_issues_enabled: true
```
Comment (VojtechBartoska) [inline .github/ISSUE_TEMPLATE/config.yml:1]: this is supposed to be false, right?

**Review by VojtechBartoska — APPROVED** [review]
LGTM.

Just 2 minor comments added.

**Comment by VojtechBartoska** [comment]
@Indastri one more thing, in the repo are 2 Pull Requests templates, we can delete one of them and it should be done in this PR as it is related.

**Comment by VojtechBartoska** [comment]
LGTM @Indastri, CI is not happy

File: .github/ISSUE_TEMPLATE/config.yml (hunk 5 above)
Comment (Indastri) [reply .github/ISSUE_TEMPLATE/config.yml]: Updated

File: .github/ISSUE_TEMPLATE/config.yml (hunk 6 above)
Comment (Indastri) [reply .github/ISSUE_TEMPLATE/config.yml]: Updated

**Review by VojtechBartoska — APPROVED** [review]

**Comment by espzav** [comment]
@Indastri LGTM. Have you got access rights for merge or should I merge it?

**Review by espzav — APPROVED** [review]

**Comment by Indastri** [comment]
@espzav if you can please merge it. Thanks!

**Pull request merged**